sklearn/feature_extraction/image.py

```diff
@@ -319,9 +319,13 @@ def extract_patches_2d(image, patch_size, max_patches=None, random_state=None):
         between 0 and 1, it is taken to be a proportion of the total number
         of patches.
 
-    random_state : int or RandomState
+    random_state : int, RandomState instance or None, optional (default=None)
         Pseudo number generator state used for random sampling to use if
         `max_patches` is not None.
+        If int, random_state is the seed used by the random number generator;
```

Comment:
In this case probably don't need to change the text, but I think it would be better if there wasn't a newline when inserting the new text,